### PR TITLE
UIDATIM-710: Cover `<LogViewer>` component with tests

### PR DESCRIPTION
**Pull request by OleksandrHladchenko1**
## Purpose
Cover LogViewer component with tests

## Approach
Covered LogViewer component with tests using Jest + RTL

## Refs
https://issues.folio.org/browse/UIDATIMP-710

## Screenshots

image

### CODE REVIEW

**Review by IvanKryzhanovskyi — APPROVED** [review]

File: src/components/LogViewer/LogViewer.test.js [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+      });
+      const instanceButton = getByText('Instance*').parentNode;
+
+      fireEvent.click(instanceButton);
+
+      expect(getByText('No record')).toBeDefined();
+    });
+  });
+
+  describe('when click on button from button group', () => {
```
Comment (mariia-aloshyna) [inline src/components/LogViewer/LogViewer.test.js:115]:
```suggestion
  describe('when clicking on button from button group', () => {
```

File: src/components/LogViewer/LogViewer.test.js [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+    describe('when click on Instance button', () => {
```
Comment (mariia-aloshyna) [inline src/components/LogViewer/LogViewer.test.js:116]: And in other places as well
```suggestion
    describe('when clicking on Instance button', () => {
```

File: src/components/LogViewer/LogViewer.test.js [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+      it('should be chosen as filter', () => {
```
Comment (mariia-aloshyna) [inline src/components/LogViewer/LogViewer.test.js:117]: And in other places as well
```suggestion
      it('Instance button should be active', () => {
```

**Review by mariia-aloshyna — COMMENTED** [review]
Add test for checking errors - when there is an error in the logs then an asterisk should be displayed next to the record type button name, `Instance*` e.g.

**Review by mariia-aloshyna — APPROVED** [review]

**Pull request merged**